tests/invite-api.js

```diff
@@ -1047,3 +1099,73 @@ function setup() {
     encryptionKeys,
   }
 }
+
+/**
+ * Assert that invites are alike, allowing for some wiggle room around
```

Comment:
can you add a note about what wiggle room means here, to help a future maintainer who might find this failing.

Reply:
Done in 8229dacf9a3ac9db495180ecea002ce83fb0e67a.